### PR TITLE
Adds the production flag -o to run_test and cleans package.json

### PR DESCRIPTION
**Pull request by edumorlom**
1. Added -o flag for production-only dependancies in run_tests.sh.
2. Cleaned package.json. All tests pass.
3. Excludes test files from npm run build in tsconfig.json.

### CODE REVIEW

File: run_test.sh [hunk 1]
```diff
@@ -178,6 +189,7 @@ while getopts pwblcsaf OPTION; do
     esac
 done
 
-if [ $OPTIND -eq 1 ]; then
```
Comment (edumorlom) [inline run_test.sh:181]: I fixed a small bug here.
According to help documentation, "no args" should run_all_tests but instead it displayed HELP.

Comment (edumorlom) [reply run_test.sh:181]: Also, bash was complaining about the ; before "then". I removed it.

Comment (shifucun) [reply run_test.sh:181]: I think the documentation is off.. the original behavior is better: do nothing if no option is provided. Can we just modify the documentation?

**Review by shifucun — COMMENTED** [review]
There are some other places need to add the -o flag:

cloudbuild.deploy.yaml
cloudbuild.screenshot.yaml

**Comment by edumorlom** [comment]
Thanks for the review, ready for re-review!

**Comment by shifucun** [comment]
thanks for update, very good cleanup!

**Review by shifucun — APPROVED** [review]

**Pull request merged**